_includes/reference/trackers_request_parameters.md

```diff
@@ -0,0 +1,6 @@
+__Parameters:__
+
+- __tracking_code__ (string), the tracking number provided by the carrier
+- __carrier__ (array), name of the carrier you want to use. Possible values are "dpd", "dhl", "gls", "hermes", "iloxx", "ups", "tnt", "fedex" and "liefery"
```

Comment:
Right. We've also added "go" yesterday. I'll introduce a value in config.yml to have the keys be stored once and used everywhere. Makes changing it easier.